docs/index.md

```diff
@@ -2636,36 +2636,121 @@ detailed information about the assertion.
+For example, for `set.mm`, propose changes to the
+[set.mm database source repository](https://github.com/metamath/set.mm).
+If it's your first time, you should contact the mailing list; they
+would be *delighted* to help you complete that process.
+
+### Feedback on this guide (not the metamath-lamp tool)
+
 If you have feedback (issues) or contributions (pull requests)
 for this *guide*, please do that by filing issues and pull requests on the
 [Metamath-lamp guide source markdown repository](https://github.com/metamath/lamp-guide/).
 Do *not* send proposals about the *tool* to this repo, as it is only for
 proposed changes to this *guide* document.
 
+### Feedback on the metamath-lamp tool (not the guide)
+
 For feedback or proposed changes to the Metamath-lamp
 *tool* itself, please file issues or create pull requests against the
 [metamath-lamp source code repository](https://github.com/expln/metamath-lamp).
 Do *not* send proposals about the *guide* to this repo, as it is only for
 proposed changes to the *tool* itself
 
-As noted earlier, if you want to run the metamath-lamp application, view the
-[Metamath-lamp application page](https://expln.github.io/lamp/latest/index.html).
 If you want to be on the bleeding edge, you can use the [development version of Metamath-lamp application page (but this may not work as expected)](https://expln.github.io/lamp/dev/index.html).
 
-If you're making a contribution to a Metamath database, such as a new
-proof, please contribute those as changes to the database's repository.
-For example, for `set.mm`, propose changes to the
-[set.mm database source repository](https://github.com/metamath/set.mm).
-If it's your first time, you should contact the mailing list; they
-would be *delighted* to help you complete that process.
+### Quick tour of the metamath-lamp source code
+
+If you wish to try to contribute *code* to the metamath-lamp
+project itself, you'll need to learn more. Here are some hints.
 
-Thank you for your time.
+The metamath-lamp program is written in the
+[ReScript programming language](https://rescript-lang.org/).
+ReScript is a robustly typed language that
+compiles to efficient and human-readable JavaScript.
+ReScript is a programming language
+similar to [OCaml](https://en.wikipedia.org/wiki/OCaml);
+it is multi-paradigm but it encourages functional programming approaches.
+ReScript source files have the extension `.res`.
+
+For more about ReScript, see this
+[introduction](https://rescript-lang.org/docs/manual/latest/introduction)
+and
+[overview](https://rescript-lang.org/docs/manual/latest/overview).
+ReScript has a sound type system and its code has no null/undefined errors.
+By default you must specify the full scoped name of a function, e.g.,
+`mytext->Js_string2.split(":")`.
+
+Metamath-lamp
+uses the widely-used React library to implement its user interface.
+React comes with a robust API to React (and JavaScript).
```

Comment:
Probably this is a typo? `React comes with a robust API to React` -> `ReScript comes with a robust API to React`